### PR TITLE
Added fpvian_basher_freestyle preset

### PR DESCRIPTION
**Pull request by FPVian**
This PR adds the "Basher Freestyle 6S 5" by FPVian" experimental preset to the Betaflight Configurator.

### CODE REVIEW

File: presets/4.3/tune/fpvian_basher_freestyle.txt [hunk 1]
```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
```
Comment (limonspb) [inline presets/4.3/tune/fpvian_basher_freestyle.txt:1]: i think its worth noting in the title if that's for 6S or 4S?
or is that for both?

File: presets/4.3/tune/fpvian_basher_freestyle.txt [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+set simplified_feedforward_gain = 140
+set simplified_pitch_pi_gain = 105
+simplified_tuning apply
+
+
+#$ OPTION BEGIN (CHECKED): Set Dynamic Idle to 30
+set dyn_idle_min_rpm = 30
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Increase Feedforward Jitter Reduction to 13
```
Comment (limonspb) [inline presets/4.3/tune/fpvian_basher_freestyle.txt:47]: is there a specific reason for jitter factor set here?
it is more of an rc_link setting and depends on what radio link person flies as far as I understand. @ctzsnooze ?

File: presets/4.3/tune/fpvian_basher_freestyle.txt [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+# Filter presets begin
+# VERY CLEAN option is only recommended if props/motors are always kept in pristine condition
+
+set gyro_lpf1_static_hz = 0
+set dyn_notch_count = 1
+set dyn_notch_q = 500
+set gyro_lpf1_dyn_min_hz = 0
+set gyro_lpf1_dyn_max_hz = 0
+set simplified_gyro_filter_multiplier = 200
+set rpm_filter_harmonics = 2
```
Comment (limonspb) [inline presets/4.3/tune/fpvian_basher_freestyle.txt:153]: i think its worth turning ON bidirectional dshot inside the option if this option requires bidirectional dshot?
And same for other filtering options, if you assume bdir is on there

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/tune/fpvian_basher_freestyle.txt [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+#$ DESCRIPTION: Well look no further! Guaranteed to contain a filter preset that works for you.
+#$ DESCRIPTION: Don't like it? Go fix your bashed-up quad!
+#$ DESCRIPTION: 
+#$ DESCRIPTION: Optimized for AUW: ~700g (including HD cam), but anything should work fine.
+#$ DESCRIPTION: The "Filtering for CLEAN builds" option is recommended for best overall performance.
+#$ DESCRIPTION:
+#$ DESCRIPTION: Leaving all filter options unchecked will leave the filters unchanged.
+#$ DESCRIPTION: You MUST enable bidirectional DSHOT300 or DSHOT600 to use the filtering options.
+#$ INCLUDE_WARNING: misc/warnings/en/rpm_filters.txt
+#$ DISCUSSION:
```
Comment (sugaarK) [inline presets/4.3/tune/fpvian_basher_freestyle.txt:20]: `#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/166`

**Comment by sugaarK** [comment]
you should include a description for the PR so people who look at the PR know what this Preset is about

File: presets/4.3/tune/fpvian_basher_freestyle.txt [hunk 5]
```diff
@@ -0,0 +1,181 @@
+#$ TITLE: Basher Freestyle 5" by FPVian
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: TUNE
+#$ STATUS: COMMUNITY
```
Comment (limonspb) [inline presets/4.3/tune/fpvian_basher_freestyle.txt:4]: we decided all the new presets become experimental first.
Then, when 3-4 pilots try them with different options from the actual Configurator tab, you can make a PR to make it community if you want.

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 1 above)
Comment (FPVian) [reply presets/4.3/tune/fpvian_basher_freestyle.txt]: Added 6S to title

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 2 above)
Comment (FPVian) [reply presets/4.3/tune/fpvian_basher_freestyle.txt]: Specified for HD Recording in Jitter Reduction Option

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 3 above)
Comment (FPVian) [reply presets/4.3/tune/fpvian_basher_freestyle.txt:156]: Moved enabling bidirectional dshot to inside each of the filtering options. Removed from DSHOT options.

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 4 above)
Comment (FPVian) [reply presets/4.3/tune/fpvian_basher_freestyle.txt]: Added PR link, thanks for the reminder!

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 5 above)
Comment (FPVian) [reply presets/4.3/tune/fpvian_basher_freestyle.txt]: Changed to experimental. How do pilots submit that they've tried the preset and it works? Comment on this PR?

**Review by ctzsnooze — APPROVED** [review]

File: presets/4.3/tune/fpvian_basher_freestyle.txt (hunk 5 above)
Comment (ctzsnooze) [reply presets/4.3/tune/fpvian_basher_freestyle.txt]: Yes that's why you include the PR link in the discussion tag

**Comment by limonspb** [comment]
@FPVian looks good now, thanks.
Probably last question - if you **want** you can make one of the dshot300/dshot600 options checked by default.
it is up to you.
For example, my freedom spec preset sets dshot600 even without any options. And then it has an option for dshot300.

I just know that most of freedom spec racers fly FCs that can support dshot 600 + RPM no problem. setting dshot300 would be a safer option tho.

But if you decide to apply dshot300/dshot600 by default, please include a second warning to:
/misc/warnings/en/dshot.txt

and you do do not want to do that - its all good we can merge now

**Comment by FPVian** [comment]
@limonspb I think I'm going to leave it as is. There will hopefully be users that use both options, and I don't want to cause someone to accidentally apply the wrong setting. Additionally, if they don't know which option to pick, they will probably be better off with BF defaults than running some of the more aggressive filter options in this preset. I could be convinced otherwise, but it doesn't sound like you have strong feelings about it.

**Review by limonspb — APPROVED** [review]

**Comment by lupuscrzy** [comment]
This preset is incredible. I have been flying with it on my 5" for a month or so now. It makes my source one frame fly like butter regardless of how beat up my props get. Other presets I've tried can be great, but once I take a couple of tumbles they are too sensitive to continue smoothly. With FPVIan's preset I maintain excellent control all the time.

I can definitively say this preset is better than the result I received with many many hours of tuning my quad myself. I'm not sure how he's done it, but this is magic!! I can't recommend it enough.

**Comment by limonspb** [comment]
> This preset is incredible. I have been flying with it on my 5" for a month or so now. It makes my source one frame fly like butter regardless of how beat up my props get. Other presets I've tried can be great, but once I take a couple of tumbles they are too sensitive to continue smoothly. With FPVIan's preset I maintain excellent control all the time.
> 
> I can definitively say this preset is better than the result I received with many many hours of tuning my quad myself. I'm not sure how he's done it, but this is magic!! I can't recommend it enough.

Thanks for the feedback!

@FPVian you can change this preset from Experimental to COMMUNITY :)

**Comment by FPVian** [comment]
Yay! Thanks for those kind words @lupuscrzy. Just submitted the PR.

**Pull request merged**